src/styles/utils/variables.scss

```
@@ -0,0 +1,9 @@
+$height-header: 60px;
+$active-link-color: #00ACDC;
+$main-color: #060b35;
+$secondary-color: #616070;
+$button-blue-color: #00acdc;
```

Why did you create two variables with the same values?
```suggestion
$active-link-color: #00ACDC;
$main-color: #060b35;
$secondary-color: #616070;
```